### ISSUE TITLE
How to handle releases built from different teams input (Mirrored from ml-gradle) ?

### ISSUE THREAD

**Issue by kosed** [marklogic/marklogic-data-hub#887]
Please see: https://github.com/marklogic-community/ml-gradle/issues/336

I am also including the issue here as DHF was my primary concern but the expected feature is also  shared with ml-gradle.

Anyone has got ideas about how to make this happen in a seamless (as possible) fashion ?

┆Status: New

**Comment by popzip** [comment]
looks like it was resolved in ml-gradle project. i'll close in the near future if no further response.

**Comment by grechaw** [comment]
We depend on a certain version of ml-gradle for the data hub plugin.  For 3.1 we will upgrade to the lastest ml-gradle.  Will that help with the shadowing issue?  We're a bit stuck extending ml-gradle for the time being, although we'd like to change that.

**Comment by kosed** [comment]
Yeah sure I understand the dependancy.
It depends on whether those quite precious mlConfigPaths/mlModulePaths are going to be available in DHF. Is it going to be ?

**Comment by grechaw** [comment]
I know it can be and don't think we need to take any action to make it so.
So a scenario to verify would be great.  Let's see if I have it right

Assuming modules are in ./extra-modules
and ./my-modules
then
mlModulePaths=./extra-modules,./my-modules

should load from both of these locations.  Would you expect these to be appended to hub-internal-config and user-config or override them?

**Comment by kosed** [comment]
Yes @grechaw example is ok for mlModulePaths and we would need an identical test case for mlConfigPaths.

Since overriding still gives you the option to provide the default folders, e.g. hub-internal-config and user-config if you are using mlConfigPaths then that is my preference.

I ran a quick tests on DHF 3.0.0 and none of the options are taken into account.
Also, talking about mlModulePaths, I can see in HubProjectImpl.java
=> this.pluginsDir = this.projectDir.resolve("plugins");

I did not see any override of this value afterwards.
